### DISCUSSION TITLE
Custom Scoring Mod

### DISCUSSION THREAD

**Discussion by MatterMr** [ppy/osu#20314] [Ideas]
A common argument for tournament events is how scoring should be handled for each event, the current solution is to standardize this system with scoreV2 in stable. Lazer has no such option, leading the scoring system to be limited to standardized or classic scoring.

(Forgive me for my generalized description of this idea, I have no technical experience on how the scoring system works and am just proposing the idea to be either shot down by moderators or to be further elaborated on by people more experience by me)

General Proposal:
My proposed solution is a custom scoring mod where elements of how a map is scored can be modified, saved as a mod preset, and allow scores to be saved only locally. This solution would be an expanded version of how the current classic mod can be modified to include modern scoring improvements. The end goal would be to allow weighted elements such as accuracy and combo to be altered for the final score calculations. This would ideally allow tournament moderators to customize how they want to weight certain elements for score evaluation or even giving a Lazer mod preset for the current scoreV2 on stable.

Ideally I just want a scoreV2 mod in lazer to allow me to practice for tournaments using that client, but a custom scoring mod seems like a more flexible solution.

There are probably community implications that could be problematic due to a split scoring, but this is not for me to consider. I am just proposing the idea.

**Comment by mk56-spn** [comment]
Sounds like a fun idea. Not sure mod would be the best place for it. Since this sounds like something playlists and multi lobbies might want to incorporate were it to become a thing
